conans/requirements.txt

```diff
@@ -11,3 +11,4 @@ pylint>=1.8.1, <1.9.0
 future==0.16.0
 pygments>=2.0, <3.0
 astroid>=1.6, <1.7
+python-gnupg>=0.4.1
```

Comment:
Why requiring this, instead of the https://pypi.python.org/pypi/gnupg (2.3.1)?

Reply:
Well, I answer myself: License: GPLv3+ is incompatible with MIT

Reply:
in addition, that other fork is the one that needs psutil